src/Illuminate/Support/Arr.php

```diff
@@ -147,14 +147,14 @@ public static function exists($array, $key)
      * Return the first element in an array passing a given truth test.
      *
      * @param  array  $array
-     * @param  callable  $callback
+     * @param  callable|null  $callback
      * @param  mixed  $default
      * @return mixed
      */
-    public static function first($array, callable $callback, $default = null)
+    public static function first($array, callable $callback = null, $default = null)
     {
         foreach ($array as $key => $value) {
-            if (call_user_func($callback, $key, $value)) {
+            if (is_null($callback) || call_user_func($callback, $key, $value)) {
```

Comment:
To avoid calling `is_null` on each iteration, you should instead use this at the beginning:
```
if (is_null($callback)) {
    return !empty($array) ? reset($array) : value($default);
}
```